examples/v0/account_management/create_customer.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example illustrates how to create a new customer under a given
+manager account.
+
+Note: this example must be run using the credentials of a Google Ads manager
+account. By default, the new account will only be accessible via the manager
+account.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+from datetime import datetime
+
+import google.ads.google_ads.client
+
+
+def main(client, customer_id):
+    customer_service = client.get_service('CustomerService')
+    resource_name = customer_service.customer_path(customer_id)
```

Comment:
This is unnecessary and actually ignored by the server.
It's confusing to have this here too, as the specified parameter should be "manager customer ID", and that should be specified when you call "customer_service.create_customer_client" only.

This object is a new one so it shouldn't have existing customer ID nor resource name yet.

Reply:
Great point, I removed this line